lib/features/children/family_history/widgets/actioned_donation_widget.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:givt_app/features/children/family_history/models/child_donation.dart';
+import 'package:givt_app/features/children/family_history/models/child_donation_helper.dart';
+import 'package:givt_app/l10n/l10n.dart';
+import 'package:givt_app/utils/app_theme.dart';
+import 'package:givt_app/utils/datetime_extension.dart';
+
+class ActionedDonationWidget extends StatelessWidget {
+  const ActionedDonationWidget(
+      {required this.donation, required this.size, super.key});
+  final ChildDonation donation;
+  final Size size;
```

Comment:
why cant we get size in the widget's build method?

Reply:
sure we can